### PR TITLE
Allow to run Guideline_Enforcer

### PR DESCRIPTION
**Pull request by cedelavergne-ledger**
Add script allowing to call the Guideline Enforcer checks from `ledger-app-workflows` repository.
Add missing packages in the container

Bump Speculos & Ragger to their latest version

### CODE REVIEW

File: dev-tools/enforcer.sh [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+#     get_app_metadata - Retrieve application metadata from manifest
+#
+#===============================================================================
+get_app_metadata() {
+    if [[ ! -f "${APP_MANIFEST}" ]]; then
+        echo "/!\ No ${APP_MANIFEST} manifest detected!"
+        echo "This file is mandatory, please add it on your repository"
+        echo "Documentation here: https://github.com/LedgerHQ/ledgered/blob/master/doc/utils/manifest.md"
+        exit 1;
+    fi
```
Comment (lpascal-ledger) [inline dev-tools/enforcer.sh:95]: I don't get why here we check for `APP_MANIFEST` existence, then later we perform operations on `${APP_DIR}/${APP_MANIFEST}`. These paths are different right? Except if `APP_DIR` is always supposed to be `./` but in that case what's the point of the check on top of the script?

Comment (cedelavergne-ledger) [reply dev-tools/enforcer.sh:95]: Good catch. This was a bad copy-paste, when retrieving the same behavior as done in ledger-app-workflows :wink:

**Comment by agrojean-ledger** [comment]
@cedelavergne-ledger shouldn't the `enforcer.sh` live on `ledger-app-workflows` and be copied from there directly (via a clone of the repo during the image build in a tmp dir) ?

**Comment by cedelavergne-ledger** [comment]
> @cedelavergne-ledger shouldn't the `enforcer.sh` live on `ledger-app-workflows` and be copied from there directly (via a clone of the repo during the image build in a tmp dir) ?
> 
> It seems to me the script is tightly linked to the workflows

No, because for the VSCode extension, we need a simple and straight forward  method, ideally based on a script: The final usage, with the extension, will be to open the `dev-tool` container, with a bash command. This latter must be as simple as possible and moreover, generic. Using a script will allow to easily maintain/improve the mechanism in the future, if needed, just by updating the container, without needing to hardcode complexe command line in the extension itself.

**Review by agrojean-ledger — APPROVED** [review]

**Pull request merged**